variants/STM32U5xx/U575A(G-I)IxQ_U585AIIxQ/variant_STEVAL_MKBOXPRO.h

```diff
@@ -0,0 +1,354 @@
+#define PB14_ALT1               (PB14 | ALT1)
+#define PB14_ALT2               (PB14 | ALT2)
+#define PB15_ALT1               (PB15 | ALT1)
+#define PB15_ALT2               (PB15 | ALT2)
+#define PC0_ALT1                (PC0  | ALT1)
+#define PC1_ALT1                (PC1  | ALT1)
+// #define PC2_ALT1                (PC2  | ALT1)
+#define PC3_ALT1                (PC3  | ALT1)
+#define PC4_ALT1                (PC4  | ALT1)
+#define PC5_ALT1                (PC5  | ALT1)
+#define PC6_ALT1                (PC6  | ALT1)
+#define PC6_ALT2                (PC6  | ALT2)
+#define PC7_ALT1                (PC7  | ALT1)
+#define PC7_ALT2                (PC7  | ALT2)
+#define PC8_ALT1                (PC8  | ALT1)
+#define PC9_ALT1                (PC9  | ALT1)
+#define PC10_ALT1               (PC10 | ALT1)
+#define PC11_ALT1               (PC11 | ALT1)
+#define PF9_ALT1                (PF9  | ALT1)
+#define PH12_ALT1               (PH12 | ALT1)
+
+#define NUM_DIGITAL_PINS        104
+#define NUM_ANALOG_INPUTS       5
+
+// On-board LED pin number
+#define LED1                    PF6
+#define LED2                    PH11
+#define LED3                    PH12
+#define LED_USB                 PF9
+#define LED_GREEN               LED1
+#define LED_RED                 LED2
+#define LED_YELLOW              LED3
+#define LED_BLUE                LED_USB
+#ifndef LED_BUILTIN
+  #define LED_BUILTIN           LED1
+#endif
+
+// On-board user button
+#define SW1_BTN                 PC13
+#ifndef USER_BTN
+  #define USER_BTN              SW1_BTN
+#endif
+
+// Timer Definitions
+// Use TIM6/TIM7 when possible as servo and tone don't need GPIO output pin
+#ifndef TIMER_TONE
+  #define TIMER_TONE            TIM6
+#endif
+#ifndef TIMER_SERVO
+  #define TIMER_SERVO           TIM7
+#endif
+
+// UART Definitions
+#ifndef SERIAL_UART_INSTANCE
+  #define SERIAL_UART_INSTANCE  4
+#endif
+
+// Default pin used for generic 'Serial' instance
+// Mandatory for Firmata
+#ifndef PIN_SERIAL_RX
+  #define PIN_SERIAL_RX         PA1
+#endif
+#ifndef PIN_SERIAL_TX
+  #define PIN_SERIAL_TX         PA0
+#endif
+
+/* Set default Wire instance on I2C1 Sensor Bus */
+#define PIN_WIRE_SDA            PB7
+#define PIN_WIRE_SCL            PB6
+
+/* Set default SPI instance on SPI2 */
+#define PIN_SPI_MOSI            PI3
+#define PIN_SPI_MISO            PI2
+#define PIN_SPI_SCK             PI1
+#define PIN_SPI_SS              PI7 // SPI_Sen_CS_A
+#define PIN_SPI_SS1             PI5 // SPI_Sen_CS_G
+
+/* SD detect signal */
+#define SD_DETECT_PIN           PC5
+/* SD Transceiver */
+#define USE_SD_TRANSCEIVER      1U
+#define SD_TRANSCEIVER_EN       PH10
+#define SD_TRANSCEIVER_SEL      PH8
+
+// Extra HAL modules
+// #if !defined(HAL_DAC_MODULE_DISABLED)
+//   #define HAL_DAC_MODULE_ENABLED
+// #endif
+// #if !defined(HAL_OSPI_MODULE_DISABLED)
+//   #define HAL_OSPI_MODULE_ENABLED
+// #endif
+#if !defined(HAL_SD_MODULE_DISABLED)
+  #define HAL_SD_MODULE_ENABLED
+#endif
+
```

Comment:
Frederic,
the MDF module should be enabled here? We need it for the audio part.
BR
Carlo

Reply:
Hi Carlo. Was not aware of that. I will add it.

Reply:
done